lib/wallet/test/unit/Cardano/Wallet/Write/Tx/Balance/TokenBundleSizeSpec.hs

```diff
@@ -228,5 +247,55 @@ instance Arbitrary (VariableSize1024 TokenBundle) where
     arbitrary = VariableSize1024 <$> resize 1024 genTokenBundle
     -- No shrinking
 
-maryTokenBundleMaxSize :: TokenBundleMaxSize
-maryTokenBundleMaxSize = TokenBundleMaxSize 4000
+instance Arbitrary Version where
+    arbitrary = arbitraryBoundedEnum
+
+data PParamsInRecentEra
+    = PParamsInBabbage (PParams StandardBabbage)
+    | PParamsInConway (PParams StandardConway)
+    deriving (Show, Eq)
+
+instance Arbitrary PParamsInRecentEra where
+    arbitrary = oneof
+        [ PParamsInBabbage <$> genPParams RecentEraBabbage
+        , PParamsInConway <$> genPParams RecentEraConway
+        ]
+
+      where
+        genPParams
+            :: RecentEra era
+            -> Gen (PParams (ShelleyLedgerEra era))
+        genPParams era = withConstraints era $ do
+            ver <- arbitrary
+            maxSize <- genMaxSize
+            return $ def
+                & ppProtocolVersionL .~ (ProtVer ver 0)
+                    -- minor version doesn't matter
+                & ppMaxValSizeL .~ maxSize
+          where
+            genMaxSize :: Gen Natural
+            genMaxSize =
+                oneof
+                -- Generate values close to the mainnet value of 4000 (and guard
+                -- against underflow)
+                [ fromIntegral . max 0 . (4000 +) <$> arbitrary @Int
+
+                -- Generate more extreme values (both small and large)
+                , fromIntegral <$> arbitrary @Word64
+                ]
+
+babbageTokenBundleSizeAssessor :: TokenBundleSizeAssessor
+babbageTokenBundleSizeAssessor = mkTokenBundleSizeAssessor RecentEraBabbage
+    $ def
+        & ppProtocolVersionL .~ (ProtVer (eraProtVerLow @StandardBabbage) 0)
+        & ppMaxValSizeL .~ maryTokenBundleMaxSize
+  where
+    maryTokenBundleMaxSize = 4000
```

Comment:
I think we should be clear about the units here:

```suggestion
    maryTokenBundleMaxSizeBytes = 4000
```
By the way, this seems to have recently changed to 5000:

![Screenshot from 2023-09-14 15-11-21](https://github.com/cardano-foundation/cardano-wallet/assets/206319/ce2f9a0e-e847-4c79-a733-56dbce6b7a3e)

Reply:
https://beta.explorer.cardano.org/en/protocol-parameters/